capabilities/remember.js

```diff
@@ -153,12 +151,10 @@ async function storeUserMemory(userId, value) {
  */
 
 async function storeUserMessage(userId, value) {
-  const { data, error } = await supabase.from("messages").insert(
-    {
-      user_id: userId,
-      value,
-    },
-  );
+  const { data, error } = await supabase.from("messages").insert({
```

Comment:
Hmmmm... So there's "storage" with embeddings, and there's "messages" without embeddings.

Might it make sense to just have one table? I think I don't understand why they are different tables.